<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogConfigMeasurement</class>
 <widget class="QDialog" name="dialogConfigMeasurement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>282</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Configure measurement - Metro</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="layoutMain">
     <item>
      <layout class="QVBoxLayout" name="layoutMeas">
       <item>
        <layout class="QHBoxLayout" name="layoutScanAmount">
         <item>
          <widget class="QLabel" name="labelPreScanAmount">
           <property name="text">
            <string>Perform this measurement</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="editScanAmount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPostScanAmount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>times</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeMeas">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>op_name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listMacros">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddScanset">
   <property name="text">
    <string>Add new scan</string>
   </property>
  </action>
  <action name="actionRemoveScanset">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionUpScanset">
   <property name="text">
    <string>Move up</string>
   </property>
  </action>
  <action name="actionDownScanset">
   <property name="text">
    <string>Move down</string>
   </property>
  </action>
  <action name="actionAddMacro">
   <property name="text">
    <string>Add new macro...</string>
   </property>
  </action>
  <action name="actionDeleteMacro">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionUpdateMacro">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialogConfigMeasurement</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogConfigMeasurement</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
